packages/calypso-e2e/src/lib/pages/gutenberg-editor-page.ts

```diff
@@ -118,12 +92,35 @@ export class GutenbergEditorPage {
 
 	/**
 	 * Dismisses the Welcome Tour (card) if it is present.
+	 *
+	 * The network request to `nux` endpoint is retrived and the value of
+	 * `show_welcome_guide` is checked. If true the action to click on the
+	 * close button is fired.
 	 */
 	async dismissWelcomeTour(): Promise< void > {
-		const frame = await this.getEditorFrame();
-		const locator = frame.locator( selectors.welcomeTourCloseButton );
+		let response;
+
+		try {
+			response = await this.page.waitForResponse( /.*nux\?_envelope.*/, {
+				timeout: 105 * 1000,
+			} );
```

Comment:
Wrapped this in a `try/catch` so that in the odd instance that either:

1. the network request is missed by this method, or;
2. the iframed editor is taking exceptionally long to load

Then simply assume the Welcome Tour doesn't exist and continue to whatever else the `waitUntilLoaded` needs to perform.